core/src/test/java/org/elasticsearch/cluster/metadata/MetaDataIndexUpgradeServiceTests.java

```diff
@@ -96,7 +97,7 @@ public void testFailUpgrade() {
             .put(IndexMetaData.SETTING_VERSION_CREATED, Version.fromString("2.1.0"))
```

Comment:
nit: Should we bump these to realistic current versions? i.e., 5.0.0 for index created and Version.V_6_0_0. minimumIndexCompatibilityVersion